src/main/java/com/exercise/tasksmanagementsystem/entity/Task.java

```diff
@@ -10,7 +12,8 @@ public class Task {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
-    private int timeSpent;
+    private Date startDate;
```

Comment:
you have to use LocalDateTime.
Because you need to calculate spent time. You can start and finish the task on the same date, but time will be different